src/databricks/labs/remorph/reconcile/query_builder/oracle_query_builder.py

```diff
@@ -0,0 +1,56 @@
+from databricks.labs.remorph.reconcile.constants import ColumnTransformationType, Constants
+from databricks.labs.remorph.reconcile.query_builder.source_query_build_adapter import (
+    SourceQueryBuildAdapter,
+)
+from databricks.labs.remorph.reconcile.recon_config import QueryColumnConfig, QueryColumnWithTransformation, Tables, \
+    Schema
+
+
+class OracleQueryBuildAdapter(SourceQueryBuildAdapter):
+
+    def __init__(self, source_type: str, table_conf: Tables, schema: list[Schema]):
+        super().__init__(source_type, table_conf, schema)
+
+    def get_select_columns(self) -> QueryColumnConfig:
+        sel_cols = [sch.column_name for sch in self.schema]
+        return QueryColumnConfig(select_cols=sel_cols, join_cols=[])
+
+    # @abstractmethod
+    # def remove_drop_cols_from_sel_cols(self) -> QueryColumnConfig:
+    #     pass
+
+    def get_join_columns(self, col_config) -> QueryColumnConfig:
+        setattr(col_config, 'join_cols', [self.table_conf.join_columns])
```

Comment:
don't use `setattr` - it's not traceable via static code analysis, use concrete assignment